### DISCUSSION TITLE
X Server to manage SO console

### DISCUSSION THREAD

**Discussion by Farcocoop** [Security-Onion-Solutions/securityonion#4190] [Unsupported Versions]
Hello. I need to access SO console remotely. My idea is using X Server. The command "xhost +" returns "xhost:  unable to open display "IP:0.0". The same happens with "export DISPALY="IP:0,0"".  Anyone knows what to do? Alternatives like VNC or RDP are good? The reason I want to access the console is to run wireshark and networkminer. Thank you

**Comment by trb143** [comment]
sshuttle over a vpn works fine.  You need to use a password and not a certificate. 
When this is running you have access to the local network via the browser.
